franka_gazebo/models/pick_tray/model.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<model>
+    <name>pick_tray</name>
+    <version>1.0</version>
+    <sdf version="1.7">model.sdf</sdf>
```

Comment:
melodic cant handle 1.7 therefore I suggest changing all sdf versions to 1.6

Reply:
Good to know!